src/mon/PGMonitor.h

```diff
@@ -38,10 +38,13 @@ class MGetPoolStats;
 
 class TextTable;
 class MPGStats;
+class PGStatService;
+class PGMapStatService;
 
 class PGMonitor : public PaxosService {
 public:
   PGMap pg_map;
+  PGMapStatService *pgservice;
```

Comment:
We can't ditch PGMonitor until post-luminous.. I suspect the most we can do once the luminous upgrade finishes is make Monitor.cc take PGMonitor out of the paxos list and/or stop sending messages to it or calling its tick().